packages/flutter_tools/lib/executable.dart

```diff
@@ -110,8 +111,10 @@ Future<void> main(List<String> args) async {
        // the build runner packages are not synced internally.
        CodeGenerator: () => const BuildRunner(),
        WebCompilationProxy: () => BuildRunnerWebCompilationProxy(),
-       // The web runner is not supported internally because it depends
+       // The web runner is not supported in google3 because it depends
        // on dwds.
        WebRunnerFactory: () => DwdsWebRunnerFactory(),
+       // The mustache dependency is different in google3
+       TemplateRenderer: () => const MustacheTemplateRenderer(),
```

Comment:
Is this a breaking change?

Reply:
I'm going to verify with a frob run, but I don't think this is actually used in google3

Reply:
https://test.corp.google.com/ui#id=OCL:297591207:BASE:297593068:1582817828420:5932081a

Should be safe to land